public/app/features/integrated-alerting/components/NotificationChannel/DeleteNotificationChannelModal/DeleteNotificationChannelModal.test.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { dataQa } from '@percona/platform-core';
+import { DeleteNotificationChannelModal } from './DeleteNotificationChannelModal';
+import { notificationChannelStubs } from '../__mocks__/notificationChannelStubs';
+
+jest.mock('../NotificationChannel.service');
+jest.mock('app/core/app_events', () => {
```

Comment:
Is it a good idea to put this mock under `app/core/app_events/__mocks__` to avoid repetition? We use this mock in many places already...

Reply:
Done :)